code/datums/ai/dog/dog_controller.dm

```diff
@@ -97,6 +97,12 @@
 /datum/ai_controller/dog/proc/on_attack_hand(datum/source, mob/living/user)
 	SIGNAL_HANDLER
 
+	var/mob/living/living_pawn = pawn
+	var/pronounce = pawn.gender == "male" ? "his" : "her"
+	var/additional_text = HAS_TRAIT(user, TRAIT_NAIVE) ? "Maybe bring a heater?" : "It looks like a corpse."
+	if(living_pawn.stat == DEAD)
+		to_chat(user, span_warning("Touching [pawn], you feel [pronounce] cold skin through the fur. [additional_text]"))
+		return
```

Comment:
```suggestion
	var/additional_text = HAS_TRAIT(user, TRAIT_NAIVE) ? "It looks like [living_pawn.p_they()] are sleeping." : "It looks like a corpse."
	if(living_pawn.stat == DEAD)
		to_chat(user, span_warning("Touching [living_pawn], you feel [living_pawn.p_their()] cold skin through the fur. [additional_text]"))
		return
```

I'd suggest you look at https://github.com/tgstation/tgstation/blob/master/code/__HELPERS/pronouns.dm since these handle grammar stuff.

Reply:
oh. What is the connection between cold skin and sleep?
Maybe leave the option with a heater?

Reply:
Mammals naturally drop in body temperature when they go to sleep, so it's not an illogical conclusion to make. I think the heater thing is cute though.

The key point about this suggestion is using `p_they` and `p_their` here, because we already naturally track pronouns on all mobs, you don't need to add the

```dm
var/pronounce = pawn.gender == "male" ? "his" : "her"
```

check in code. Much cleaner to do it as tim does it above.